controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ControllerMaintenance.java

```diff
@@ -118,13 +118,13 @@ public Intervals(SystemName system) {
             this.outstandingChangeDeployer = duration(3, MINUTES);
             this.versionStatusUpdater = duration(3, MINUTES);
             this.readyJobsTrigger = duration(1, MINUTES);
-            this.deploymentMetricsMaintainer = duration(5, MINUTES);
+            this.deploymentMetricsMaintainer = duration(6, MINUTES);
             this.applicationOwnershipConfirmer = duration(12, HOURS);
-            this.systemUpgrader = duration(1, MINUTES);
-            this.jobRunner = duration(90, SECONDS);
+            this.systemUpgrader = duration(90, SECONDS);
+            this.jobRunner = duration(105, SECONDS);
```

Comment:
```suggestion
            this.jobRunner = duration(90, SECONDS);
```
This one probably times out sometimes because it does something synchronously that rarely takes a long time — it's just supposed to dispatch to an executor. I'll fix that instead.